### PR TITLE
chore: update nv24 documents with upgrade epochs

### PR DESCRIPTION
**Pull request by rjan90**
chore: update nv24 documents with upgrade epochs

### CODE REVIEW

File: upgrades.json [hunk 1]
```diff
@@ -687,14 +687,48 @@
       ],
       "activation": {
         "mainnet": {
-          "epoch": null,
-          "time": "2024-11-20T00:00:00Z",
-          "actors": {}
+          "epoch": 4461240,
+          "time": "2024-11-20T23:00:00Z",
+          "actors": {
+            "account": "bafk2bzacecia5zacqt4gvd4z7275lnkhgraq75shy63cphakphhw6crf4joii",
```
Comment (rvagg) [inline upgrades.json:693]:
```suggestion
            "_manifest": "bafy2bzaceakwje2hyinucrhgtsfo44p54iw4g6otbv5ghov65vajhxgntr53u",
            "account": "bafk2bzacecia5zacqt4gvd4z7275lnkhgraq75shy63cphakphhw6crf4joii",
```

File: upgrades.json [hunk 2]
```diff
@@ -687,14 +687,48 @@
...
+            "system": "bafk2bzacecvcqje6kcfqeayj66hezlwzfznytwqkxgw7p64xac5f5lcwjpbwe",
+            "verifiedregistry": "bafk2bzacecudaqwbz6dukmdbfok7xuxcpjqighnizhxun4spdqvnqgftkupp2"
+          }
         },
         "calibrationnet": {
           "epoch": 2078794,
           "time": "2024-10-23T13:30:00Z",
-          "actors": {}
+          "actors": {
+            "account": "bafk2bzacecwdkoknhok52hlddoetdkqfwohhv4bx6csu3x6o7aduryv5ufssu",
```
Comment (rvagg) [inline upgrades.json:715]:
```suggestion
            "_manifest": "bafy2bzaceax5zkysst7vtyup4whdxwzlpnaya3qp34rnoi6gyt4pongps7obw",
            "account": "bafk2bzacecwdkoknhok52hlddoetdkqfwohhv4bx6csu3x6o7aduryv5ufssu",
```

**Comment by rjan90** [comment]
I do not have merge permissions in this repo, so could you click on the merge button for me @rvagg ?

**Comment by rjan90** [comment]
Pushed an update which removes `FIP-0086: Fast Finality` from the nv24 scope, given that the F3-activation has been postponed until certain bug-fixes surfaced has been resolved: https://github.com/filecoin-project/community/discussions/74#discussioncomment-11164349

**Review by rvagg — APPROVED** [review]

**Pull request merged**